prac_06/guitar.py

```diff
@@ -0,0 +1,26 @@
+"""
+CP1404 Prac 6 Jack Kerlin
+estimate (for guitar and guitar_test): 15 min
+actual: 14 min
+"""
+
+
+class Guitar:
+    """Represent a guitar object"""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost:.2f}"
+
+    def get_age(self):
+        return 2023 - self.year
```

Comment:
You could possibly make "2023" a constant such as "CURRENT_YEAR" so that it is easier to change it in the future.